### PR TITLE
PKG-4186

### PR DESCRIPTION
**Pull request by cbouss**
Update to 0.1.7.post0

https://github.com/osqp/qdldl-python/compare/v0.1.7...v.0.1.7.post0

### CODE REVIEW

**Review by SC426 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,9 +33,10 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - libqdldl {{ version }}
+    - libqdldl 0.1.7
```
Comment (danpetry) [inline recipe/meta.yaml:37]: when this gets updated next, someone might not know to update the pinning here. How about a comment or a jinja function that splits the version?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,9 +33,10 @@ requirements:
...
     - numpy {{ numpy  }}
-    - pybind11 2.10.1
+    - pybind11 2.12.0
+    - pybind11-abi
```
Comment (danpetry) [inline recipe/meta.yaml:40]: is this a mutex? does it need to be put everywhere that pybind is used?
also it's only built for linux-64 AFAICT, not sure why the other platforms are passing

Comment (cbouss) [reply recipe/meta.yaml:40]: pybind11-abi is noarch. It is indeed a mutex, which should be used everywhere pybind11 is used. It didn't matter much until recently, as the abi was at version 4 for an extended time. However now both abi 4 and 5 exist.

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:37]: Thanks, I added a comment

File: recipe/meta.yaml (hunk 2 above)
Comment (danpetry) [reply recipe/meta.yaml:40]: The ABI is mutually exclusive anyway, though no? i.e. 5 on win or py >= 3.12, 4 otherwise?

Comment (danpetry) [reply recipe/meta.yaml:40]: and even if they weren't, not sure why a mutex would be needed as you'd have two package variants, which conda wouldn't install together? AFAIU a mutex is more like, there are two different packages with different names and we only want one of them at a time

**Pull request merged**